### PR TITLE
Don't humanize the URLs

### LINKED ISSUE #83: humanizeUrl is redundant. Creates invalid URI
I was using this generator and noticed that it strictly turns valid `URI` to normal string without no `http`. I don't know if this was intentional but I looked through the code and seems like it's just a redundant package and creates an extra URL warning. 
There is already a `normalizeUrl` package. We can use that instead of `humanizeUrl`

issue

### PR DESCRIPTION
**Pull request by RocktimSaikia**
Fixes #83 
Removes the package `humanizeUrl`. 
Uses `normalizeUrl` instead.

### CODE REVIEW

File: app/templates/_package.json [hunk 1]
```diff
@@ -18,26 +19,33 @@
...
 		"index.js"<% if (cli) { %>,
-		"cli.js"<% } %>
+			"cli.js"<%
+		} %>
 	],
 	"keywords": [<% if (cli) { %>
 		"cli-app",
-		"cli",<% } %>
+			"cli",<%
+		} %>
```
[the remark this answers is not in the document]
Comment (RocktimSaikia) [reply app/templates/_package.json]: @sindresorhus ping

[the remark this answers is not in the document]
Comment (sindresorhus) [reply app/templates/_package.json]: > My vscode has Prettier installed

You should disable this when working on external projects ;)

File: app/index.js [hunk 2]
```diff
@@ -89,7 +88,7 @@ module.exports = class extends Generator {
 				name: this.user.git.name(),
 				email: this.user.git.email(),
 				website: props.website,
-				humanizedWebsite: humanizeUrl(props.website),
+				normalizedWebsite: props.website,
```
Comment (sindresorhus) [inline app/index.js:91]: This is just the same as the `website` item.

File: app/templates/_package.json [hunk 3]
```diff
@@ -40,4 +40,4 @@
 			"text"
 		]
 	}<% } %>
-}
+}
```
Comment (sindresorhus) [inline app/templates/_package.json:43]: ^

**Comment by sindresorhus** [comment]
The PR title should better explain what problem the PR solves.

File: app/templates/_package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (RocktimSaikia) [reply app/templates/_package.json]: Thanks. I'll keep that in mind. I'm new to open source so these tips are really helpful for me👍

File: app/index.js (hunk 2 above)
Comment (RocktimSaikia) [reply app/index.js]: @sindresorhus  Right. fixed it in the lastest commit :)

File: app/templates/_package.json (hunk 3 above)
Comment (RocktimSaikia) [reply app/templates/_package.json]: What is this `^` mean ?

Comment (sindresorhus) [reply app/templates/_package.json]: It's an arrow pointing out:

> Don't do unrelated changes.

Comment (sindresorhus) [reply app/templates/_package.json]: Always look over the full diff after every change to catch mistakes like this.

Comment (RocktimSaikia) [reply app/templates/_package.json]: @sindresorhus thanks for the advice 👍

**Pull request merged**